Packaging on Windows doesn't work with latest MSYS2

I have all SDL2 and other DLLs installed in the recent MSYS2, when packaging I get
`
vladi@DESKTOP-0N1N2MK MINGW64 ~/chocolate-doom/pkg/win32
$ make
mkdir staging-doom
LC_ALL=C ./cp-with-libs --ldflags="-lmingw32 -mwindows -lSDL2main -lSDL2  -lsamplerate  -lpng16  -lwinmm" \
               ../../src/chocolate-doom.exe staging-doom

Missing DLLs not found in path []:
        libpng16-16.dll
        SDL2.dll
        SDL2_net.dll
        SDL2_mixer.dll
        libsamplerate-0.dll
make: *** [GNUmakefile:40: staging-doom] Error 1
`

Also when I put an IWAD file in the /src/ directory with the built exes, they won't run and give messages of the missing DLLs instead.

P.S. Building with CMake I have the same issue.

--- a/pkg/win32/GNUmakefile
+++ b/pkg/win32/GNUmakefile
@@ -37,9 +37,9 @@ hook-strife: staging-strife
 
 staging-%:
 	mkdir $@
-	LC_ALL=C ./cp-with-libs --ldflags="$(LDFLAGS)" \
+	LC_ALL=C ./cp-with-libs --ldflags="$(LDFLAGS)" --dll_path="$(MSYSTEM_PREFIX)/bin" \
 	               $(TOPLEVEL)/src/$(PROGRAM_PREFIX)$*.exe $@
-	LC_ALL=C ./cp-with-libs --ldflags="$(LDFLAGS)" \
+	LC_ALL=C ./cp-with-libs --ldflags="$(LDFLAGS)" --dll_path="$(MSYSTEM_PREFIX)/bin" \
 	               $(TOPLEVEL)/src/$(PROGRAM_PREFIX)setup.exe \
 	               $@/$(PROGRAM_PREFIX)$*-setup.exe
 

